cdp_scrapers/instances/seattle.py

```diff
@@ -72,14 +72,11 @@ def __init__(self):
         )
 
         try:
+            url = "https://seattlechannel.org/"
             urlopen("https://seattlechannel.org/")
```

Comment:
Since we've defined `url`, we should replace this to `urlopen(url)`

Reply:
Oops. Of course. Made the change.